nomos-services/network/src/backends/libp2p/command.rs

```diff
@@ -25,7 +25,7 @@ pub enum Command {
     StreamSend {
         peer_id: PeerId,
         protocol: StreamProtocol,
-        packet_body: PacketBody,
+        data: Box<dyn AsyncWritable + Send + Sync>,
```

Comment:
This indeed is much better, but, I think you actually may want `Sink`? https://docs.rs/tokio/latest/tokio/io/struct.Sink.html